server/src/grpc_server.rs

```diff
@@ -38,14 +45,17 @@ impl<OB: QuoteBook> GrpcServer<OB> {
         msg_bus_tx: Sender<Msg>,
         quote_book: OB,
         client_listen_uri: DeqsClientUri,
+        quote_minimum_tuples: Vec<(TokenId, u64)>,
```

Comment:
I think its better to have this just be a Hashmap and let the caller worry about producing it. This makes it the callers responsibility to ensure the same token id does not appear twice in the array, which can happen and right now will be silently swallowed.